infra/resources/_modules/web_apps/functions_etl.tf

```
@@ -0,0 +1,43 @@
+module "etl_func" {
+  source = "github.com/pagopa/dx//infra/modules/azure_function_app?ref=f339355788f12e5e4719159dca45d7c0b5c0c537"
+
+  environment = merge(var.environment, {
+    app_name        = "etl",
+    instance_number = "01"
+  })
+
+  application_insights_connection_string   = var.application_insights.connection_string
+  application_insights_sampling_percentage = 100
+
+  tier = "test"
```

```suggestion
  tier = "m"
```

we can safely use the `m` size for this function